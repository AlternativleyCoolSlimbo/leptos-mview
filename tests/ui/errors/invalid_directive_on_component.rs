use leptos::*;
use leptos_mview::view;

fn style() {
    view! {
        Component style:color="white";
    }
}

fn class() {
    view! {
        Component class:red={true};
    }
}

#[component]
fn Component() -> impl IntoView {
    view! {
        button;
    }
}

fn main() {}
