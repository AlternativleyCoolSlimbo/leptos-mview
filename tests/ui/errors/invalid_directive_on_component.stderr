error: directive `style` is not supported on components
 --> tests/ui/errors/invalid_directive_on_component.rs:6:19
  |
6 |         Component style:color="white";
  |                   ^^^^^^^^^^^

error: directive `class` is not supported on components
  --> tests/ui/errors/invalid_directive_on_component.rs:12:19
   |
12 |         Component class:red={true};
   |                   ^^^^^^^^^
